### PR TITLE
how-to: add offline cluster upgrade method in ansible doc

### PR DESCRIPTION
**Pull request by ericsyh**
### What is changed, added or deleted? 

Fix PRM-104 to provide introduction how to upgrade the cluster in offline condition.

### Which version does your change affect? 

dev/v3.0/v2.1

### CODE REVIEW

File: dev/how-to/upgrade/from-previous-version.md [hunk 1]
```diff
@@ -179,7 +179,9 @@ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
 ansible-playbook local_prepare.yml
 ```
 
-## 滚动升级 TiDB 集群组件
+## 集群在线滚动升级
+
+### 滚动升级 TiDB 集群组件
 
 如果当前 `process_supervision` 变量使用默认的 `systemd` 参数，则通过 `excessive_rolling_update.yml` 滚动升级 TiDB 集群。
```
Comment (TomShawn) [inline dev/how-to/upgrade/from-previous-version.md]: 是否应该加上 `使用以下命令来滚动升级 TiDB 集群组件：`？

File: dev/how-to/upgrade/from-previous-version.md [hunk 2]
```diff
@@ -201,10 +203,32 @@ ansible-playbook rolling_update.yml
 >
 > 为优化 TiDB 集群组件的运维管理，TiDB 3.0 版本对 `systemd` 模式下的 `PD service` 名称进行了调整。在升级到 TiDB 3.0 版本后，滚动升级及日常滚动重启 TiDB 集群统一使用 `rolling_update.yml` 操作，不再使用 `excessive_rolling_update.yml`。
 
-## 滚动升级 TiDB 监控组件
+### 滚动升级 TiDB 监控组件
```
Comment (TomShawn) [inline dev/how-to/upgrade/from-previous-version.md]: ditto. 加上 `使用以下命令来滚动升级 TiDB 监控组件：`？

File: dev/how-to/upgrade/from-previous-version.md [hunk 3]
```diff
@@ -201,10 +203,32 @@ ansible-playbook rolling_update.yml
...
 
 ```bash
 ansible-playbook rolling_update_monitor.yml
 ```
+
+## 集群离线升级
+
+### 关闭集群
+
+此操作会按顺序关闭整个 TiDB 集群所有组件（包括 PD、TiDB、TiKV 等组件和监控组件）。
```
Comment (TomShawn) [inline dev/how-to/upgrade/from-previous-version.md]:
```suggestion
执行关闭集群操作会按顺序关闭整个 TiDB 集群内的所有组件，包括 PD、TiDB、TiKV 等组件和监控组件。执行命令如下：
```

File: dev/how-to/upgrade/from-previous-version.md [hunk 4]
```diff
@@ -201,10 +203,32 @@ ansible-playbook rolling_update.yml
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+ansible-playbook stop.yml
+```
+
+### 离线升级集群
+
+重新部署 TiDB 集群软件
```
Comment (TomShawn) [inline dev/how-to/upgrade/from-previous-version.md]:
```suggestion
执行以下命令来重新部署 TiDB 集群软件：
```

File: dev/how-to/upgrade/from-previous-version.md (hunk 1 above)
Comment (ericsyh) [reply dev/how-to/upgrade/from-previous-version.md]: 在标题上加有点太长了，我在内容里加吧

**Comment by TomShawn** [comment]
Please address the comments in v2.1 and v3.0. Rest LGTM.

**Comment by ericsyh** [comment]
@TomShawn Done. PTAL.

**Comment by TomShawn** [comment]
LGTM, please involve a technical review @ericsyh

**Comment by ericsyh** [comment]
@liubo0127 PTAL

**Comment by liubo0127** [comment]
The 2.1 version of the documentation may not require the steps to upgrade to 3.0.

**Comment by lilin90** [comment]
> The 2.1 version of the documentation may not require the steps to upgrade to 3.0.

@ericsyh What do you think?

**Comment by TomShawn** [comment]
@ericsyh When the content is ready, please also update v3.1 as well, thanks~

**Comment by ericsyh** [comment]
Will add documents for v3.1 upgrade in this PR.

**Pull request closed without merging**